### PR TITLE
screenshot of app on error

### PR DESCRIPTION
**Pull request by bruno-garcia**
image

### CODE REVIEW

File: src/SymbolCollector.Android/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -4,4 +4,4 @@
 	<uses-permission android:name="android.permission.INTERNET" />
 	<uses-permission android:name="android.permission.WAKE_LOCK" />
 	<application android:allowBackup="true" android:label="@string/app_name" android:supportsRtl="true" android:theme="@style/AppTheme" />
-</manifest>
+</manifest>
```
Comment (bruno-garcia) [inline src/SymbolCollector.Android/Properties/AndroidManifest.xml:7]:
```suggestion
</manifest>

```

File: src/SymbolCollector.Android/MainActivity.cs [hunk 2]
```diff
@@ -397,5 +406,35 @@ protected override void Dispose(bool disposing)
...
+            public ScreenshotAttachment()
+                : this(
+                AttachmentType.Default,
+                new ScreenshotAttachmentContent(),
+                "screenshot",
+                 "image/png")
+            {
+
+            }
+            private ScreenshotAttachment(
```
Comment (lucas-zimerman) [inline src/SymbolCollector.Android/MainActivity.cs:421]: Could you add an extra line here?

File: src/SymbolCollector.Android/MainActivity.cs [hunk 3]
```diff
@@ -397,5 +406,35 @@ protected override void Dispose(bool disposing)
...
+        private class ScreenshotAttachment : Attachment
+        {
+            public ScreenshotAttachment()
+                : this(
+                AttachmentType.Default,
+                new ScreenshotAttachmentContent(),
+                "screenshot",
+                 "image/png")
+            {
+
```
Comment (lucas-zimerman) [inline src/SymbolCollector.Android/MainActivity.cs:419]: and remove this one?

**Review by lucas-zimerman — APPROVED** [review]

File: src/SymbolCollector.Android/MainActivity.cs (hunk 3 above)
Comment (bruno-garcia) [inline src/SymbolCollector.Android/MainActivity.cs:419]:
```suggestion
```

Comment (lucas-zimerman) [inline src/SymbolCollector.Android/MainActivity.cs]:
```suggestion

```

File: src/SymbolCollector.Android/MainActivity.cs (hunk 2 above)
Comment (bruno-garcia) [inline src/SymbolCollector.Android/MainActivity.cs:420]:
```suggestion

            private ScreenshotAttachment(
```

Comment (lucas-zimerman) [inline src/SymbolCollector.Android/MainActivity.cs:421]:
```suggestion

            private ScreenshotAttachment(
```

**Pull request merged**